geoindicators/src/test/resources/org/orbisgis/orbisprocess/geoclimate/geoindicators/lczTests/expectedLcz.geojson

```diff
@@ -0,0 +1,8 @@
+{
+"type": "FeatureCollection",
+"name": "expectedLcz",
+"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:EPSG::2154" } },
```

Comment:
Rename the file to expectedLcz_french_metropole.geojson  because 2154 epsg is only for this area